### PR TITLE
Add Jenkinsfile CD Dev

### PR DESCRIPTION
**Pull request by datanel**
need to be tested, help @pascal-vs

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            steps {
+                script {
+                    docker.withRegistry('', 'kisiodigital-user-dockerhub') {
+                        sh "make release"
+                    }
+                }
+                script {
+                    def handle = triggerRemoteJob job: 'pad_deploy', parameters: 'ENVIRONMENT=dev\nVERSION=latest', preventRemoteBuildQueue: true, remoteJenkinsName: 'jenkins-deployment', useCrumbCache: true, useJobInfoCache: true
+                    echo 'Remote Status: ' + handle.getBuildStatus().toString()
+                    // I don't know what to do with handle
```
Comment (woshilapin) [inline Jenkinsfile:42]: Can it fail? Should we handle any kind of exception in the triggered job?

**Review by woshilapin — DISMISSED** [review]
Really nice, I'm really happy to see this kind of CD happening. Curious to see how it will unfold.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                script {
+                    def handle = triggerRemoteJob(
+                        remoteJenkinsName: 'jenkins-deployment',
+                        job: 'pad_deploy',
+                        parameters: 'ENVIRONMENT=dev\nVERSION=latest',
+                        blockBuildUntilComplete: true,
+                        preventRemoteBuildQueue: true,
+                    )
+                    echo 'Remote Status: ' + handle.getBuildStatus().toString()
+                }
```
Comment (pascal-vs) [inline Jenkinsfile:48]: Is it possible to have only one `script` block?
```suggestion
                script {
                    docker.withRegistry('', 'kisiodigital-user-dockerhub') {
                        sh "make release"
                    }
                    def handle = triggerRemoteJob(
                        remoteJenkinsName: 'jenkins-deployment',
                        job: 'pad_deploy',
                        parameters: 'ENVIRONMENT=dev\nVERSION=latest',
                        blockBuildUntilComplete: true,
                        preventRemoteBuildQueue: true,
                    )
                    echo 'Remote Status: ' + handle.getBuildStatus().toString()
                }
```

**Review by pascal-vs — APPROVED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (datanel) [reply Jenkinsfile]: done

File: Jenkinsfile (hunk 1 above)
Comment (datanel) [reply Jenkinsfile]: I don't know the behaviour. I can't test until I have a token to trigger the remote job. wait and see

**Review by pascal-vs — APPROVED** [review]

**Pull request merged**